### PR TITLE
Enable stackdriver tracing by default

### PR DESCRIPTION
**Pull request by gargnupur**

### CODE REVIEW

File: asm/cluster/istio-operator.yaml [hunk 1]
```diff
@@ -37,3 +37,6 @@ spec:
       multiCluster:
         # Provided to ensure a human readable name rather than a UUID.
         clusterName: "PROJECT_ID/us-central1-c/asm-cluster" # {"$ref":"#/definitions/io.k8s.cli.substitutions.cluster-name"}
+      enableTracing: true
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply asm/cluster/istio-operator.yaml:40]: @mandarjog, @bianpengyuan: do we have any existing docs about potential quota issues?

**Comment by gargnupur** [comment]
@maasen : can you please help merge this?

**Comment by maasen** [comment]
@elfinhe can you take a look & merge?

**Comment by elfinhe** [comment]
I've reverted the PR because it broke the injection. Feel free to create a new one after the issue is fixed.

**Pull request merged**